packages/adapter-vercel/files/reroute.js

```diff
@@ -0,0 +1,67 @@
+import { reroute } from 'HOOKS';
```

Comment:
I've seen `SERVER` and `MANIFEST` in most adapters, but we could standardise it to `__NAME__`

Reply:
Given that `reroute.js`  gets bundled with `esbuild` anyways wouldn't it be preferrable to use an import-alias instead of doing a string replacement? Seems like it would be more robust.

Reply:
Can you expand on what an import alias is? I’m unfamiliar with esbuild